### PR TITLE
[move-prover] Improve README.md

### PR DESCRIPTION
**Pull request by shazqadeer**
## Motivation

Improve README.md for prover installation.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

Manual testing

### CODE REVIEW

**Comment by shazqadeer** [comment]
@emmazzz : Please take a look.

File: language/move-prover/scripts/README.md [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
-change the ownership of these directories to your user.
-  sudo chown -R $(whoami) /usr/local/bin /usr/local/lib /usr/local/sbin
-And make sure that your user has write permission.
-  chmod u+w /usr/local/bin /usr/local/lib /usr/local/sbin
+# Installation steps
 
-The commands `boogie` and `z3` should be in your path once these scripts have finished successfully.
+1. The installation of Dotnet Core could fail on your machine because a few directories
+   must be writable by your user account.  You should change the ownership of these
+   directories to your user.
```
Comment (emmazzz) [inline language/move-prover/scripts/README.md:10]: Maybe use positive words here? To ensure the successful installation of Dotnet Core, you should change the ownership of these directories...

File: language/move-prover/scripts/README.md [hunk 2]
```diff
@@ -1,32 +1,39 @@
...
+3. ./install-boogie.sh
+4. ./install-z3.sh
 
-To check the location of the tools installed, use the following commands:
+Troubleshooting:
+On bash, the commands `boogie` and `z3` should be in your path once these scripts
+have finished successfully.  On zsh, you may have to add /Users/$(whoami)/.dotnet/tools
+manually to $PATH.
+
+# Running Move Prover
```
Comment (emmazzz) [inline language/move-prover/scripts/README.md:23]: I don't know what a better title would be but this section doesn't really teach me how to run Move Prover.

Comment (junkil-park) [reply language/move-prover/scripts/README.md:23]: Environment variable setting?

File: language/move-prover/scripts/README.md (hunk 1 above)
Comment (shazqadeer) [reply language/move-prover/scripts/README.md]: Done.  Take a look.

File: language/move-prover/scripts/README.md (hunk 2 above)
Comment (shazqadeer) [reply language/move-prover/scripts/README.md]: Done. Take a look.

**Pull request closed without merging**